config/default.schema.json

```diff
@@ -135,33 +135,59 @@
 			"default": "feedback@hpi-schul-cloud.de",
 			"description": "Contact E-Mail Adress to The HPI School-Cloud team."
 		},
+		"MATRIX_MESSENGER": {
+			"type": "object",
+			"description": "Matrix messenger properties, required always to be defined",
+			"required": [
+				"URI",
+				"SERVERNAME",
+				"SECRET",
+				"STUDENT_ROOM_CREATION"
+			],
+			"properties": {
+				"URI": {
+					"type": "string",
+					"format": "uri",
+					"default": "https://matrix.messenger.schule",
+					"description": "The URI of the Matrix Messenger server."
+				},
+				"EMBED_URI": {
+					"type": "string",
+					"format": "uri",
+					"pattern": ".*(?<!/)$",
+					"description": "Where to find the messenger dependencies."
+				},
+				"SERVERNAME": {
+					"type": "string",
+					"default": "messenger.schule",
+					"description": "Servername of the Matrix Messenger server."
+				},
+				"SECRET": {
+					"type": "string",
```

Comment:
Please mark it as secret value otherwise it is displayed in log by server start and show later im shd.

Reply:
> For security reasons, by default all values of string typed properties having secret or key in their name will be replaced with a hash-id

As described in https://github.com/hpi-schul-cloud/commons#secrets